### PR TITLE
Fix lint errors staging/src/k8s.io

### PR DESCRIPTION
**Pull request by baasbank**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug

/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
This PR fixes golint errors in 
    - staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1
    - staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller 
    - staging/src/k8s.io/sample-apiserver/pkg/cmd/server
    - staging/src/k8s.io/sample-apiserver/pkg/apiserver

**Which issue(s) this PR fixes**:

Fixes part of #68026

**Special notes for your reviewer**:
:)
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ import (
 	flunderstorage "k8s.io/sample-apiserver/pkg/registry/wardle/flunder"
 )
 
+// Scheme - defines a new runtime scheme
```
Comment (neolit123) [inline staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go]: please remove the `-` from all the comments.
them comments them self seem fine.

File: staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go [hunk 2]
```diff
@@ -74,8 +77,8 @@ type completedConfig struct {
 	ExtraConfig   *ExtraConfig
 }
 
+// CompletedConfig - Embeds a private pointer that cannot be instantiated outside of this package.
```
Comment (neolit123) [inline staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go]: lowercase `embeds`.

File: staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go [hunk 3]
```diff
@@ -37,6 +37,7 @@ func Resource(resource string) schema.GroupResource {
 	return SchemeGroupVersion.WithResource(resource).GroupResource()
 }
 
+// SchemeBuilder - initializes a scheme builder
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
```
Comment (neolit123) [inline staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go]: should the comment be above `SchemeBuilder` here?

File: staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go [hunk 4]
```diff
@@ -37,6 +37,7 @@ func Resource(resource string) schema.GroupResource {
...
 	AddToScheme   = SchemeBuilder.AddToScheme
```
Comment (neolit123) [inline staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go]: possibly needs a comment too?

**Review by neolit123 — COMMENTED** [review]
thanks for the PR @baasbank 
please keep the commits squashed to one.

/priority backlog

**Comment by spiffxp** [comment]
/hold
Please update your PR description to say "Part of #68026" instead of "Fixes".  This doesn't close out the issue in its entirety.

**Comment by baasbank** [comment]
> /hold
> Please update your PR description to say "Part of #68026" instead of "Fixes". This doesn't close out the issue in its entirety.

@spiffxp Done.

File: staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go (hunk 1 above)
Comment (baasbank) [reply staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go]: All `-` have been removed.

File: staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go (hunk 2 above)
Comment (baasbank) [reply staging/src/k8s.io/sample-apiserver/pkg/apiserver/apiserver.go]: Fixed.

File: staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go (hunk 3 above)
Comment (baasbank) [reply staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go]: Moved the comment to the line just before.

File: staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go (hunk 4 above)
Comment (baasbank) [reply staging/src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1/register.go]: Added.

**Review by neolit123 — COMMENTED** [review]
/lgtm
/ok-to-test

**Comment by spiffxp** [comment]
/hold cancel
/approve
for the hack/ changes

**Comment by spiffxp** [comment]
/assign @sttts 
for the /approve, looks like you cover both of the staging dirs

**Pull request merged**